docs/whatsnew.rst

```diff
@@ -93,82 +93,56 @@ space's name wiki".
 Technical novelties in eXo Platform 5.1
 ========================================
 
-**Upgrades**
+**New features**
 
-In the version 5.0, many tools/applications used by eXo Platform were 
-upgraded to newer  versions in order to benefit from their latest 
-updates and improvements: 
+In the version 5.1, we integrated a new feature which is the push notifications on mible.
+This new feature is supported by eXo mobile applications. It relies on Firebase Cloud Messaging 
+and a new addon packaged by default in the Enterprise Edition (exo-push-notifications).
```

Comment:
I don't think we should talk about Push Notifications in technical new features, it is already listed in functional new features.